src/renderer/components/ParticipantsTile/ParticipantList.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <ul>
+    <li class="field" v-for="(item, index) in list" v-bind:key="index">
+      <b-tag
+        size="is-medium"
+        attached
+        closable
+        aria-close-label="Remove participant"
+        @close="onRemove(index)"
+      >{{index + 1}}. {{item}}</b-tag>
+    </li>
+  </ul>
+</template>
+
+<script>
+export default {
+  props: {
+    list: { type: Array },
+  },
+  methods: {
+    onRemove(index) {
+      this.list.splice(index, 1);
```

Comment:
This operation should be done in parent component (the "owner" of participants list) via events. Or ideally in Vuex - the simplest and most readable solution.